docs/sources/tutorials/ha_setup.md

```diff
@@ -30,7 +30,8 @@ Currently alerting supports a limited form of high availability. Since v4.2.0, a
 
 ## User sessions
 
-> Beginning with Grafana v6.0 and above the following only applies when using [Auth Proxy Authentication](/auth/auth-proxy/).
+> After Grafana 6.2 you don't need to configure session storage since the database will be used by default.
+> If you want to offload the login session data from the database you can configure [remote_cache]({{< relref "configuration.md" >}}#remote_cache)
```

Comment:
```suggestion
> If you want to offload the login session data from the database you can configure [remote_cache]({{< relref "configuration.md" >}}#remote-cache)
```

Reply:
I don't have access to commit this suggestion FYI